src/vmod_vsthrottle.vcc

```diff
@@ -95,6 +95,44 @@ Example
 		}
 
 
+$ABI vrt
+$Function VOID return_token(STRING key, INT limit, DURATION period,
+                         DURATION block=0)
+
+Arguments:
+
+  - key: A unique identifier to define what is being throttled - more examples below
+  - limit: How many requests in the specified period
+  - period: The time period
+  - block: a period to deny all access after meeting the threshold
```

Comment:
these three arguments are already documented in 3 places, I think they should be explained once and referenced in the function documentation

Reply:
Makes sense!

Fixed with 2ad675e and 4c24ec7